cpp/bench/ann/src/common/benchmark.hpp

```diff
@@ -268,24 +267,22 @@ void bench_search(::benchmark::State& state,
         state.SkipWithError(std::string(e.what()));
       }
 
-      auto end = std::chrono::high_resolution_clock::now();
-
-      auto elapsed_seconds = std::chrono::duration_cast<std::chrono::duration<double>>(end - start);
       // advance to the next batch
       batch_offset = (batch_offset + n_queries) % query_set_size;
       queries_processed += n_queries;
-      state.SetIterationTime(elapsed_seconds.count());
```

Comment:
I have removed this because it is only useful, if we call `UseManualTime()` while adding the benchmark.  We could keep this and switch to ManualTime. This would more or less agree with the current `Time` column, and would have the same issues.